src/main/java/org/folio/circulation/domain/UpdateRequestService.java

```diff
@@ -46,6 +49,16 @@ public UpdateRequestService(RequestRepository requestRepository,
         .thenApply(r -> r.next(requestNoticeSender::sendNoticeOnRequestUpdated))));
   }
 
+  private Result<Request> refuseWhenPatronCommentChanged(
+    Request updated, Result<Request> originalRequestResult) {
+
+    return originalRequestResult.failWhen(
+      original -> succeeded(!Objects.equals(updated.getPatronComments(), original.getPatronComments())),
```

Comment:
Can the `Objects.equals` method be statically imported?

```suggestion
      original -> succeeded(!equals(updated.getPatronComments(), original.getPatronComments())),
```

Reply:
No, unfortunately. It overlaps with `Object.equals`

Reply:
Shucks